app/controllers/shopping_list_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ShoppingListController < ApplicationController
+  def index
+    @recipes = Recipe.where(user_id: current_user.id)
+    @ingredients = RecipeFood.where(recipe_id: @recipes.map(&:id))
+    @foods = Food.where(id: @ingredients.map(&:food_id))
+
+    @total_price = ShoppingList.new.total_price(@ingredients)
+  end
+end
```

Comment:
- [ ] Kindly add a create action for a shopping list to be generated, currently, when I click to generate a 
shopping list, there is no form for a new list. See the image below.

![shopping_list](https://user-images.githubusercontent.com/53124548/163595422-0a5ddff2-2a68-435c-9d20-a5705d8d64e4.png)

Reply:
Thank you for your observations, changes will be made as requested.